app/src/main/java/com/firebase/uidemo/database/ChatActivity.java

```diff
@@ -70,7 +71,10 @@ protected void onCreate(Bundle savedInstanceState) {
         mSendButton = (Button) findViewById(R.id.sendButton);
         mMessageEdit = (EditText) findViewById(R.id.messageEdit);
 
-        mChatRef = FirebaseDatabase.getInstance().getReference().child("chats");
+        mEmptyListView = findViewById(R.id.emptyTextView);
+
+        mRef = FirebaseDatabase.getInstance().getReference();
+        mChatRef = mRef.child("chats");
```

Comment:
Yeah, I split into two refs and apparently forgot to initialize one when resolving the conflict. It was my first time using the web conflict resolution editor and I must've borked something. I'll fix it tomorrow back at my desktop.